frontend/src/app/shared/components/autocompleter/project-menu-autocomplete/project-menu-autocomplete.component.ts

```diff
@@ -145,6 +150,24 @@ export class ProjectMenuAutocompleteComponent {
     this.dropModalOpen = false;
   }
 
+  onKeydown(event:KeyboardEvent):void {
+    if (event.keyCode === KeyCodes.ENTER) {
+      this.handleKeyEnter(event);
```

Comment:
This results in **always** selecting the first element no matter whether you already navigated with the arrows to another project or not.

Reply:
@HDinger Fixed here: https://github.com/opf/openproject/pull/11014/files#diff-cac016dd915b59a29fc6e15bc8b658a4f6a131b726b90b5489c511de38f4fd14R165